src/Joonasw.AspNetCore.SecurityHeaders/Hpkp/HpkpOptions.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Collections.Generic;
 
-namespace Joonasw.AspNetCore.SecurityHeaders
+namespace Joonasw.AspNetCore.SecurityHeaders.Hpkp
```

Comment:
Please change this namespace back and the file to its original location, avoiding breaking changes.